### PR TITLE
Fix LFBGA354 and WLCSP72 land patterns

### PR DESCRIPTION
**Pull request by m-hilgendorf**
LFBGA354:

(notice: mirrored)

lfbga354-doc
lfbga354-corrected

WLCSP72:
wlcsp72-doc
wlcsp72

### CODE REVIEW

**Review by PhilippeFerreiraDeSousa — DISMISSED** [review]
If you checked it is correct, then it looks good to me

**Comment by bpellenard** [comment]
Are the overlaps ok on the second picture?

**Comment by m-hilgendorf** [comment]
That's controlled by the solder mask registration design rule, which is too large by default for this land pattern. It is not controlled in this source code.

**Review by bpellenard — CHANGES_REQUESTED** [review]
The first one is wrong, there is a gap of two in the rows.

File: components/st-microelectronics/landpatterns.stanza [hunk 1]
```diff
@@ -29,9 +29,19 @@ public pcb-landpattern LFBGA144:
...
 public pcb-landpattern LFBGA354:
-  defn depop? (r:Int, c:Int):
-    (c == 19 and contains?([5, 15], r)) or
-    (c == 16 and contains?([6, 8, 10, 12, 14], r))
+  defn depop? (r:Int, c:Int) :
+    contains?(
+      [
+        [4  18],
+        [14 18],
+        [5  17],
```
Comment (bpellenard) [inline components/st-microelectronics/landpatterns.stanza:37]: [5, 15]

Comment (m-hilgendorf) [reply components/st-microelectronics/landpatterns.stanza]: Fixed!

**Comment by m-hilgendorf** [comment]
Good catch, fixed
lfbga354-fixed
:

**Review by bpellenard — APPROVED** [review]
Looks good to me!

**Pull request merged**